Source/ui/UI.js

```diff
@@ -48,6 +48,146 @@ export function alert(title, text) {
   return dialog.runModal()
 }
 
+export const INPUT_TYPE = {
+  string: 'string',
+  slider: 'slider',
+  selection: 'selection',
+  // coming soon
+  // number: 'number',
+  // color: 'color',
+  // path: 'path'
+}
+
+export function getInputFromUser(messageText, options, callback) {
```

Comment:
I chose to merge the different input methods into a single one so to reduce the API surface and make sure we have the same signature for all of them (the selection input method was super weird).

The result is now provided in a callback so that we can support making that modal a sheet in the future (needs to be async) and so that the fact the user cancels is very clear (error)